src/main/java/org/opentripplanner/middleware/controllers/api/MonitorTripController.java

```diff
@@ -0,0 +1,61 @@
+package org.opentripplanner.middleware.controllers.api;
+
+import com.mongodb.client.model.Filters;
+import org.bson.conversions.Bson;
+import org.eclipse.jetty.http.HttpStatus;
+import org.opentripplanner.middleware.models.MonitoredTrip;
+import org.opentripplanner.middleware.persistence.Persistence;
+import spark.Request;
+
+import static com.mongodb.client.model.Filters.eq;
+import static org.opentripplanner.middleware.auth.Auth0Connection.isAuthorized;
+import static org.opentripplanner.middleware.auth.Auth0Connection.isValidUser;
+import static org.opentripplanner.middleware.utils.JsonUtils.logMessageAndHalt;
+
+/**
+ * Implementation of the {@link ApiController} abstract class for managing monitored trips. This controller connects
+ * with Auth0 services using the hooks provided by {@link ApiController}.
+ */
+public class MonitorTripController extends ApiController<MonitoredTrip> {
+    private static final long MAXIMUM_ALLOWED_MONITORED_TRIPS = 5;
```

Comment:
I don't mind having this as a constant for now. I defer to the other reviewers.

Reply:
Let's make this be obtained from the application config.